src/linux/agent/systemui.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 var ApplicationInfo, RunningAppProcessInfo, RunningTaskInfo, GET_META_DATA;
-var context, packageManager, activityManager;
+var context, packageManager, activityManager, Intent, ComponentName;
```

Comment:
These should go on the line before, which are declarations for the imports/types, right after `ApplicationInfo`, so the imported types are in alphabetical order. (And reordered below to be assigned in the same order.)